src/bit_vec.rs

```diff
@@ -15,8 +15,10 @@
 //! `BitVec` specific serialization.
 
 use bitvec::{
-	vec::BitVec, store::BitStore, order::BitOrder, slice::BitSlice, boxed::BitBox, mem::BitMemory
+	vec::BitVec, store::BitStore, order::BitOrder, slice::BitSlice, boxed::BitBox,
 };
+use bitvec::macros::internal::funty::IsNumber;
```

Comment:
This is what `cargo` suggested to resolve the Problem that `let element_bits = T::Mem::BITS as u32;` (line 100) doesn't work - and I couldn't find any good alternative in the docs. I am not convinced this is the best way, but not sure how to solve this otherwise...

Reply:
Nope. The solution is still to fix this upstream. And we will clearly not import something that has the name `internal`in it.